README.adoc

```diff
@@ -37,7 +37,8 @@ You will learn how to build a MicroProfile Rest Client to access remote RESTful
 You'll update the template interface for a MicroProfile Rest Client, which maps to the remote service that you want to call, to use the `CompletionStage` return type. 
```

Comment:
suggested edit:

You'll update the template interface for a MicroProfile Rest Client to use the `CompletionStage` return type. 

The template interface maps to the remote service that you want to call.